packages/ui/tabs/src/Tabs.tsx

```diff
@@ -0,0 +1,83 @@
+import { createContext } from "@layer-lib/react-context";
+import { HTMLAttributes, useState } from "react";
+
+interface TabsContextType {
+  defaultValue: string;
+  value?: string;
+  onValueChange: (val: string) => void;
+}
+
+const [TabsProvider, useTabsContext] = createContext<TabsContextType>();
+
+interface TabsProps extends React.PropsWithChildren, TabsContextType {}
+
+const Tabs = ({ defaultValue, onValueChange, children }: TabsProps) => {
+  const [value, setValue] = useState(defaultValue);
+  return (
+    <TabsProvider
+      defaultValue={defaultValue}
+      value={value}
+      onValueChange={(value: string) => {
+        onValueChange(value);
+        setValue(value);
+      }}
+    >
+      {children}
+    </TabsProvider>
+  );
+};
+
+interface TabsListProps extends HTMLAttributes<HTMLDivElement> {}
+
+const TabsList = ({ children, ...props }: TabsListProps) => {
+  return (
+    <div aria-label="tabs-navigation" {...props}>
```

Comment:
```suggestion
    <div role="tablist" aria-label="tabs-navigation" {...props}>
```

![image](https://github.com/user-attachments/assets/a3e7a032-3227-4f31-9c98-c15e9be8a90a)